@here/harp-mapview/lib/text/TextElementGroupState.ts

```diff
@@ -127,17 +131,22 @@ export class TextElementGroupState {
             return this.m_textElementStates;
         }
 
-        // Compute the sortPriority once for all elements, because the computation is done more
-        // than once per element. Also, make sorting stable by taking the index into the array into
-        // account, this is required to get repeatable results for testing.
+        // Do the actual sort based on view distance.
+        this.m_textElementStates.sort((a: TextElementState, b: TextElementState) => {
+            // Move elements with undefined view distance to the end.
+            if (a.viewDistance === b.viewDistance) {
```

Comment:
this you don't need, because if they are the same, the subtraction below will return 0

Reply:
even if they're both undefined? that's why I have that there.

Reply:
your right, of course